README.md

```diff
@@ -4,15 +4,45 @@ The samples in this repository demonstrate the various capabilities of the [Temp
 
 If you want to learn more about the Temporal Server and how it works, read the documentation at [https://docs.temporal.io](https://docs.temporal.io).
 
-## Setup
+## Docker Compose setup
+**1. Download the repository.**
+```bash
+$ git clone git@github.com:temporalio/samples-php.git
+$ cd samples-php
+```
+
+**2. Build docker images.**
+```bash
+$ docker-compose build 
+```
+
+**3. Start server and application containers.**
+```bash
+$ docker-compose up
+```
+
+**4. Run a sample**
+
+To run a sample in docker use:
 
+```bash
+$ docker-compose exec app php app.php {sample-name}
+```
+
+To observe active workers:
+
+```bash
+$ docker-compose exec app rr workers -i
+```
+
+## Local Setup
```

Comment:
What is the difference between Docker Compose Setup and Local Setup?

Seems like there are now two different approaches to running these sample?

Reply:
Docker setup basically one file to run. Local - install PHP, GRPC and etc. So it's a bit harder. I would presume we can easily remove the info about local setup and keep only Docker one.

Reply:
I think having both paths is totally cool! 
I suggest we just call out this difference above -

Reply:
i added info about the core differences, feel free to change it